src/settings/SettingsPage.tsx

```diff
@@ -22,6 +22,7 @@ export interface CopilotSettings {
   googleApiKey: string;
   openRouterAiApiKey: string;
   defaultModelKey: string;
+  defaultTag: string;
```

Comment:
How about `defaultConversationTag` and put it below the `defaultSaveFolder` below